### PR TITLE
build fix

### PR DESCRIPTION
**Pull request by amponce**
Fixes build 

```
Page                                                      Size     First Load JS
┌ ● / (1184 ms)                                           4.2 kB          186 kB
├   /_app                                                 0 B            94.6 kB
├ ● /[...slug] (3104014 ms)                               3.77 kB         185 kB
├   ├ /central-arkansas-health-care/stories (3134 ms)
├   ├ /southeast-louisiana-health-care/stories (3079 ms)
├   ├ /salt-lake-city-health-care/stories (3056 ms)
├   ├ /salem-health-care/stories (3013 ms)
├   ├ /alexandria-health-care/stories (3004 ms)
├   ├ /philadelphia-health-care/stories (2989 ms)
├   ├ /western-colorado-health-care/stories (2984 ms)
├   └ [+1510 more paths] (avg 2042 ms)
├ ○ /404                                                  181 B          94.8 kB
├ λ /api/exit-preview                                     0 B            94.6 kB
├ λ /api/preview                                          0 B            94.6 kB
├ ● /demo/audienceTopics (791 ms)                         3.79 kB         185 kB
├ ● /demo/banner (1512 ms)                                4.51 kB         125 kB
├ ● /demo/bannerAlert (796 ms)                            5.62 kB         137 kB
├ ● /demo/blockAlert (731 ms)                             4.63 kB         125 kB
├ ● /demo/button (784 ms)                                 382 B           174 kB
├ ● /demo/emailContacts (831 ms)                          388 B           174 kB
├ ● /demo/expandableText (855 ms)                         394 B           174 kB
├ ● /demo/linkTeasers (769 ms)                            447 B           174 kB
├ ● /demo/media (970 ms)                                  3.82 kB         185 kB
├ ● /demo/pagination/exampleA/[page] (13313 ms)           3.86 kB         118 kB
├   ├ /demo/pagination/exampleA/9 (716 ms)
├   ├ /demo/pagination/exampleA/4 (705 ms)
├   ├ /demo/pagination/exampleA/11 (705 ms)
├   ├ /demo/pagination/exampleA/7 (704 ms)
├   ├ /demo/pagination/exampleA/6 (696 ms)
├   ├ /demo/pagination/exampleA/0 (689 ms)
├   ├ /demo/pagination/exampleA/3 (689 ms)
├   └ [+13 more paths] (avg 647 ms)
├ ● /demo/pagination/exampleB/[page] (12284 ms)           4.63 kB         119 kB
├   ├ /demo/pagination/exampleB/5 (666 ms)
├   ├ /demo/pagination/exampleB/7 (666 ms)
├   ├ /demo/pagination/exampleB/2 (646 ms)
├   ├ /demo/pagination/exampleB/0 (638 ms)
├   ├ /demo/pagination/exampleB/9 (626 ms)
├   ├ /demo/pagination/exampleB/12 (626 ms)
├   ├ /demo/pagination/exampleB/16 (621 ms)
├   └ [+13 more paths] (avg 600 ms)
├ ● /demo/partials/benefitHubsLinks (959 ms)              3.37 kB          98 kB
├ ● /demo/personProfiles (1570 ms)                        1.39 kB         175 kB
├ ● /demo/richTextCharLimit1000 (921 ms)                  396 B           174 kB
├ ● /demo/table (959 ms)                                  6.83 kB         188 kB
└ ● /demo/wysiwyg (1014 ms)                               386 B           174 kB
+ First Load JS shared by all                             94.6 kB
  ├ chunks/framework-51b5d01fa1eb82cf.js                  45.1 kB
  ├ chunks/main-ffb4ad0633e0e3b4.js                       29.1 kB
  ├ chunks/pages/_app-3e07450cf9053e99.js                 17.9 kB
  ├ chunks/webpack-626e1822fdf64276.js                    2.44 kB
  └ css/e0d4433917d86b11.css                              484 B

λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
○  (Static)  automatically rendered as static HTML (uses no initial props)
●  (SSG)     automatically generated as static HTML + JSON (uses getStaticProps)
```

### CODE REVIEW

File: src/mocks/mockMediaQuery.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const MatchMedia = Object.defineProperty(window, 'matchMedia', {
```
Comment (penny-lischer) [inline src/mocks/mockMediaQuery.ts:1]: I dont think you meant to check this in, or did you?

File: src/pages/demo/bannerAlert/index.tsx [hunk 2]
```diff
@@ -36,22 +35,20 @@ export async function getStaticProps(
     'field_banner_alert_vamcs',
     'field_banner_alert_vamcs.field_office',
   ])
-  params.addFilter('field_banner_alert_vamcs', null, 'IS NOT NULL')
```
Comment (penny-lischer) [inline src/pages/demo/bannerAlert/index.tsx:39]: How is this object being returned now?

Comment (amponce) [reply src/pages/demo/bannerAlert/index.tsx:39]: for some reason when we leave in line 39 and generate all 1500 banners we get into a call-stack loop partway though, I messed with this for about an hour on Friday and finally resolved the build it by removing this line altogether. The include on line 36 seems to cover the object we need back but adding the null check causes problems and I don't really understand why

File: src/mocks/mockMediaQuery.ts (hunk 1 above)
Comment (amponce) [reply src/mocks/mockMediaQuery.ts]: Removed, thanks for spotting that

**Pull request merged**